test/unit/config/config-formatters.test.js

```diff
@@ -5,175 +5,146 @@
-    falseyValues.forEach((val) => {
-      t.test(`should map ${val} to false`, (t) => {
-        t.equal(formatters.boolean(val), false)
-        t.end()
+    for (const val of falseyValues) {
+      await t.test(`should map ${val} to false`, () => {
+        assert.equal(formatters.boolean(val), false)
       })
-    })
+    }
 
     // these are new tests but do not want to change behavior of this formatter
     // but anything that is not a falsey value above is true ¯\_(ツ)_/¯
     const truthyValues = ['true', 'anything-else', '[]', '{}']
-    truthyValues.forEach((val) => {
-      t.test(`should map ${val} to true`, (t) => {
-        t.equal(formatters.boolean(val), true)
-        t.end()
+    for (const val of truthyValues) {
+      await t.test(`should map ${val} to true`, () => {
+        assert.equal(formatters.boolean(val), true)
       })
-    })
+    }
   })
 
-  tap.test('object', (t) => {
-    t.autoend()
-
-    t.test('should parse json string as an object', (t) => {
+  await test('object', async (t) => {
+    await t.test('should parse json string as an object', () => {
       const val = '{"key": "value"}'
       const result = formatters.object(val)
-      t.same(result, { key: 'value' })
-      t.end()
+      assert.deepStrictEqual(result, { key: 'value' })
     })
 
-    t.test('should log error and return null if it cannot parse option as json', (t) => {
+    await t.test('should log error and return null if it cannot parse option as json', () => {
       const loggerMock = { error: sinon.stub() }
       const val = 'invalid'
-      t.notOk(formatters.object(val, loggerMock))
-      t.equal(loggerMock.error.args[0][0], 'New Relic configurator could not deserialize object:')
-      t.match(loggerMock.error.args[1][0], /SyntaxError: Unexpected token/)
-      t.end()
+      assert.ok(!formatters.object(val, loggerMock))
+      assert.equal(
+        loggerMock.error.args[0][0],
+        'New Relic configurator could not deserialize object:'
+      )
+      assert.match(loggerMock.error.args[1][0], /SyntaxError: Unexpected token/)
     })
   })
 
-  tap.test('objectList', (t) => {
-    t.autoend()
-
-    t.test('should parse json string a collection with 1 object', (t) => {
+  await test('objectList', async (t) => {
+    await t.test('should parse json string a collection with 1 object', () => {
       const val = '{"key": "value"}'
       const result = formatters.objectList(val)
-      t.same(result, [{ key: 'value' }])
-      t.end()
+      assert.deepStrictEqual(result, [{ key: 'value' }])
     })
 
-    t.test('should log error and return null if it cannot parse option as json', (t) => {
+    await t.test('should log error and return null if it cannot parse option as json', () => {
       const loggerMock = { error: sinon.stub() }
       const val = 'invalid'
-      t.notOk(formatters.objectList(val, loggerMock))
-      t.equal(
+      assert.ok(!formatters.objectList(val, loggerMock))
```

Comment:
`assert.notOk` exists. But I would write:
```suggestion
      assert.equal(formatters.objectList(val, loggerMock), null)
```